charts/GPP-publicatiebank/Chart.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: gpp-publicatiebank
+description: A helm chart for the ICATT Openbaar-Documenten-Publicatie-Component
```

```suggestion
description: A helm chart for the ICATT GPP-Publicatiebank
```
we've settled on the component names now ;)